upstream-community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sysdig-operator.v1.4.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Security, Monitoring"
+    certified: "false"
+    description: |-
+      Sysdig is a unified platform for container and microservices monitoring, troubleshooting, security and forensics. Sysdig platform has been built on top of Sysdig tool and Sysdig Inspect open-source technologies.
+    containerImage: docker.io/sysdiglabs/sysdig-operator:1.4.0
+    createdAt: 2019-03-08T18:30:00Z
+    support: Sysdig, Inc.
+    repository: https://github.com/sysdiglabs/sysdig-operator/
+    alm-examples: |-
+      [{
+        "apiVersion": "sysdig.com/v1alpha1",
+        "kind": "SysdigAgent",
+        "metadata": {
+          "name": "basic-agent-deployment"
+        },
+        "spec": {
+          "sysdig": {
+            "accessKey": "XXX"
+          }
+        }
+      },
+      {
+        "apiVersion": "sysdig.com/v1alpha1",
+        "kind": "SysdigAgent",
+        "metadata": {
+          "name": "agent-with-ebpf-and-secure-enabled"
+        },
+        "spec": {
+          "ebpf": {
+            "enabled": true
+          },
+          "secure": {
+            "enabled": true
+          },
+          "sysdig": {
+            "accessKey": "XXX"
+          }
+        }
+      }]
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: SysdigAgent
+      name: sysdigagents.sysdig.com
+      version: v1alpha1
+  description: |-
+    [Sysdig](https://www.sysdig.com/) is a unified platform for container and microservices monitoring, troubleshooting, security and forensics. Sysdig platform has been built on top of [Sysdig tool](https://sysdig.com/opensource/sysdig/) and [Sysdig Inspect](https://sysdig.com/blog/sysdig-inspect/) open-source technologies.
+
+    This operator installs the Sysdig Agent for [Sysdig Monitor](https://sysdig.com/product/monitor/) and [Sysdig Secure](https://sysdig.com/product/secure/) to all nodes in your cluster via a DaemonSet.
```

Comment:
How does a user hook this up to their sysdig account?

Can you mention any standard configs or parameters that might either be required or are popular? Similar to https://github.com/operator-framework/community-operators/blob/master/upstream-community-operators/postgresql/postgres-operator.v3.5.0.clusterserviceversion.yaml#L35 or https://github.com/operator-framework/community-operators/blob/master/upstream-community-operators/percona/percona-xtradb-cluster-operator.0.3.0-rc1.clusterserviceversion.yaml#L140

Reply:
Sure! I have added an example and explained all parameters.

Thanks for the feedback! :grinning: